src/python/library/requirements/requirements_grpc.txt

```diff
@@ -24,7 +24,11 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-protobuf>=3.5.0
+# Restrict maximum version due to breaking protobuf 4.21.0 changes
+# (see https://github.com/protocolbuffers/protobuf/issues/10051)
+# TODO: Update to be compatible wtih 4.0 release
+# https://jirasw.nvidia.com/browse/DLIS-3809
```

Comment:
nit: combine these lines to something like this, don't need full jira link (see other DLIS references in codebase).
```
# TODO: Update to be compatible wtih 4.0 release (DLIS-3809)
```

Reply:
Thanks, Ryan. Updated.